torch/_logging/_registrations.py

```diff
@@ -45,6 +45,11 @@
     "aot_joint_graph",
     "Print FX joint graph from AOTAutograd, prior to partitioning. Useful for debugging partitioning",
 )
+register_artifact(
+    "post_grad_graphs",
+    "Prints the FX graph generated by post grad passes. Useful to understand what's being given to Inductor after post grad passes",
+    visible=True,
```

Comment:
Do we want to make it visible by default (perhaps that's a bad name)? That just shows it by default when you do `TORCH_LOGS="help"`

Reply:
output_code and aot_graphs are visible, and this is no different. I can change it if this isn't as important or as useful.

Reply:
Yeah it's filtered by "how useful do we expect this to be to the average PyTorch user".

This feels like a bit more of a dev detail, so perhaps better to not show it by default. To be clear, this has no impact on the actual usage/what's shown, it's only about what's shown in the help message.